ci/run_fuzzit.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -eux
```

Comment:
Do you really want -x here? This might expose credentials.

Reply:
no credentials are passed in this file (it is taken straight from the environment variables. I removed the need for fuzzit auth ${FUZZIT_API_KEY))